.betterer.eslint.config.js

```diff
@@ -111,4 +112,20 @@ module.exports = [
       '@grafana/no-untranslated-strings': 'error',
     },
   },
+  {
+    name: 'grafana/alerting-betterer',
+    files: ['public/app/features/alerting/**/*.{ts,tsx,js,jsx}'],
+    plugins: {
+      'react-refresh': reactRefresh,
+    },
+    rules: {
+      'react-refresh/only-export-components': [
+        'error',
+        {
+          allowConstantExport: true,
```

Comment:
Vite supports constant exports in its HMR, so we can allow this (based on docs in https://www.npmjs.com/package/eslint-plugin-react-refresh)